index.js

```diff
@@ -0,0 +1,57 @@
+function calculate(array) {
+  let cleanArray = []
+
+  array.forEach(element => {
+    if (element === '-' || element === '/' || element === '*' || element === '+') {
+      cleanArray.push(element)
+    } else if (element === null) {
+      cleanArray.push(0)
+    } else if (element === '') {
+      return ''
+    }
+    else if (isNaN(element) === false) {
```

Comment:
This is nicely done. A similar common way to write this is:

`!isNaN(element)`

We usually skip the `=== false` since the return value of is an os boolean and a complete predicate on it's own.

No changes needed. Just sharing related info.